docs/data-sources/dli_datasource_connections.md

```diff
@@ -0,0 +1,110 @@
+---
+subcategory: "Data Lake Insight (DLI)"
+---
+
+# huaweicloud_dli_datasource_connections
+
+Use this data source to get the list of DLI enhanced datasource connections within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "name" {}
+
+data "huaweicloud_dli_datasource_connections" "test" {
+  name = var.name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `name` - (Optional, String) Specifies the name of the datasource connection.
+
+* `tags` - (Optional, Map) Specifies the key/value pairs of the datasource connections.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The data source ID.
+
+* `connections` - The list of the datasource connections.  
+  The [connections](#datasource_connections) structure is documented below.
+
+<a name="datasource_connections"></a>
+The `connections` block supports:
+
+* `created_at` - The creation time of the datasource connection.
+
+* `hosts` - List of the user-defined hosts information.
+  The [hosts](#datasource_connections_hosts) structure is documented below.
+
+* `id` - The ID of the datasource connection.
```

Comment:
it's better to make `id` as the first attribute in a block.

Reply:
fixed